packages/components/src/mobile/global-styles-context/index.native.js

```diff
@@ -3,7 +3,12 @@
  */
 import { createContext, useContext } from '@wordpress/element';
 
-const GlobalStylesContext = createContext( { style: {} } );
+/**
+ * Internal dependencies
+ */
+import styles from '../../../../block-library/src/style.native.scss';
+
+const GlobalStylesContext = createContext( styles );
```

Comment:
Need to refactor GlobalStylesContext with https://github.com/WordPress/gutenberg/blob/master/packages/edit-site/src/components/editor/global-styles-provider.js